ubccr/xdmod
# Update Dashboard Center Counts

## Description
- Missed adding this `roleGrid.updateCenterCounts()` in `SectionExistingUsers.js` in the last PR.
  - This will ensure that the Center Counts are up to date each time a user is loaded.
- Also resolved the issue of the 'Status' label not being updated due to there being two elements with the same 'id'. This was done by having the two classes that instantiated their own instances of `XDMoD.AdminPanel` to instead share one instance ( in `XDMoD.UserManagement.Panel` )

## Motivation and Context
The UI should display the correct information to the user. 

## Tests performed
Manual Tests: 
**Test 1: **
- Login to the Internal Dashboard
- Navigate to 'User Management' -> 'XDMoD Account Requests'
- Click 'Create & Manage Users'
- Select the 'Current Users'  Tab
- Click on a few users in the default group ( 'Demo Users' ). 
  - Ensure that the Center Count displayed is correct
- Select the 'Internal Users' group
- Click on a few users and ensure that the Center Count is displayed correctly. 
- Click 'Close'
- Click 'Create & Manage Users' 
- Click around on a few more users and ensure that the Center Count is displayed correctly.

**Test 2: **
- Login to the Internal Dashboard
- Navigate to 'User Management' -> 'Existing Users' -> 
- Click 'Create & Manage Users'
- Select the 'Current Users'  Tab
- Click on a few users in the default group ( 'Demo Users' ). 
  - Ensure that the Center Count displayed is correct
  - Ensure that the 'status' is displayed correctly
- Select the 'Internal Users' group
- Click on a few users and ensure that the Center Count is displayed correctly. 
- Click 'Close'
- Click 'Create & Manage Users' 
- Click around on a few more users and ensure that the Center Count / status text is displayed correctly.

**Test 3: **
- Login to the Internal Dashboard
- Navigate to 'User Management' -> 'Existing Users' 
- Double click on an existing User account
- Ensure that:
  - the 'User Management' modal has popped up
  - that the user you double clicked is selected
  - that the Center Counts are displayed correctly.
  - that the 'status' is displayed correctly.
- Click on a few more users in the 'User Management' Modal
- Ensure that: 
  - the Center Counts are displayed correctly.
  - Ensure that the 'status' is displayed correctly.
 
## Types of changes


## Checklist:
